clients/src/main/java/org/apache/kafka/clients/admin/ConsumerGroupDescription.java

```diff
@@ -108,7 +113,8 @@ public ConsumerGroupDescription(String groupId,
                                     GroupState groupState,
                                     Node coordinator,
                                     Set<AclOperation> authorizedOperations) {
-        this(groupId, isSimpleConsumerGroup, members, partitionAssignor, GroupType.CLASSIC, groupState, coordinator, authorizedOperations);
+        this(groupId, isSimpleConsumerGroup, members, partitionAssignor, GroupType.CLASSIC, groupState, coordinator, authorizedOperations,
+            Optional.empty(), Optional.empty());
     }
 
     public ConsumerGroupDescription(String groupId,
```

Comment:
Was this constructor introduced in 4.0 only? It seems to be the case but we need to double check. If it is not, we would need to introduce a new one in order to not break existing usages.

Reply:
This constructor is introduced in 4.0. It is not existent in 3.9.

https://github.com/apache/kafka/blob/84caaa6e9da06435411510a81fa321d4f99c351f/clients/src/main/java/org/apache/kafka/clients/admin/ConsumerGroupDescription.java#L45-L81